zetaclient/metrics/metrics.go

```diff
@@ -170,6 +170,27 @@ var (
 		Name:      "num_connected_peers",
 		Help:      "The number of connected peers (authenticated keygen peers)",
 	})
+
+	// SchedulerTaskInvocationCounter tracks invocations categorized by status, group, and name
+	SchedulerTaskInvocationCounter = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: ZetaClientNamespace,
+			Name:      "scheduler_task_invocations_total",
+			Help:      "Total number of task invocations",
+		},
+		[]string{"status", "task_group", "task_name"},
+	)
+
+	// SchedulerTaskExecutionDuration measures the execution duration of tasks
+	SchedulerTaskExecutionDuration = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: ZetaClientNamespace,
+			Name:      "scheduler_task_duration_seconds",
+			Help:      "Histogram of task execution duration in seconds",
+			Buckets:   []float64{0.05, 0.1, 0.2, 0.3, 0.5, 1, 1.5, 2, 3, 5, 7.5, 10, 15}, // 50ms to 15s
```

Comment:
`prometheus.DefBuckets` is probably fine

Reply:
I think it's not suitable in this case because most likely the latency will come from RPC networking which is definitely more than`5ms`

```go
// 5ms to 10s
var DefBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10}
```